CONTRIBUTING.md

```diff
@@ -83,6 +83,19 @@ made targeting the `main` branch, while larger `refactor` pull requests target
 
 Pull requests require two approvals before merging.
 
+### Releases
+
+Twilight uses `git-cliff` and `cargo-release` for releases. These are both
+available as binaries from `crates.io`. They are executed via
+`gen-changelogs.sh` and `release.sh` respectively. Invoking `release.sh` with
+extra arguments passes them to the saved `cargo-release` command.
+
+Steps to create a patch release across the workspace:
+- generate the changelogs: `$ ./gen-changelogs.sh`
+- customize the changelogs as necessary, and commit them
+- preview the release process: `$ ./release.sh patch -vv`
```

Comment:
I think the full list of available bump levels arguments should be written to avoid looking at the `cargo-release` documentation. 

```suggestion
- preview the release process: `$ ./release.sh [level] -vv` (where level is `patch`, `minor` or `major`)
```